Userland/Libraries/LibGfx/TIFFGenerator.py

```diff
@@ -615,6 +616,8 @@ def main():
 
     output_path = Path(args.output)
 
+    os.makedirs(output_path)
```

Comment:
(And a Python nitpick if you really think this is the way to go)
Please use the pathlib library. From the top of my head, that should work:
```python
output_path.mkdir(exist_ok=True)
```

Reply:
I'm trying to use lagom out of the serenity source tree.  Doing this makes it clone serenity into your project, and as part of the build it seems to copy the parts it needs to build into _deps.  It is during this process the missing directory is a problem.

I've made a starter app which shows my setup, which has this issue: https://github.com/alankemp/lagom-starter-app